app/views/recipes/index.html.haml

```diff
@@ -1,12 +1,14 @@
 .recipes.bg-grey
-  .center 
+  .center
     %h1 Recipes
     %p= link_to 'New Recipe', new_recipe_path
 
-    - @recipes.each do |recipe|
-      .recipe
-        %h3= link_to recipe.title, recipe_path(recipe)
-        %p= recipe.description
-        %p
-          Posted by:
-          = mail_to recipe.user.email
+- @categories.each do |category|
+  %h3= category.name
+  - category.recipes.each do |recipe|
```

Comment:
This will list all categories on the index, with related recipes beneath.
As per our LoFi, the  plan was to have the links to view one category at the time.
But we can do this in a separate story.